setup.py

```diff
@@ -22,7 +22,7 @@
     'sphinxcontrib-qthelp',
     'Jinja2>=2.3',
     'Pygments>=2.0',
-    'docutils>=0.14,<0.18',
+    'docutils>=0.16,<0.19',
```

Comment:
Why do you drop 0.14 and 0.15?

Reply:
I suppose not strictly required. 0.13 and 0.12 were dropped two years ago (https://github.com/sphinx-doc/sphinx/commit/b2360a133b1d18e7f300daf94daaa7fa65c4f260) and I thought reasonable to keep the supported versions fairly recent. 

I could split that to another PR if you'd prefer, though?

A

Reply:
Could you revert it, please? Basically, we use the current LTS of Ubuntu to choose the support versions of libraries and interpreters.

Reply:
Sorry -- I saw this https://www.sphinx-doc.org/en/master/internals/release-process.html#python-version-support-policy for Python support but nothing on Docutils. Do you plan to drop support for several versions at once in April 2023 (next Python bump)?

Will revert.

A

Reply:
Yes. It would be better to keep compatibility until 2023. But, Of course, we can discuss upgrading if really needed.